src/components/number/number.spec.js

```diff
@@ -6,8 +6,22 @@ import Label from "../../__internal__/label";
 import { assertStyleMatch } from "../../__spec_helper__/test-utils";
 import InputPresentation from "../../__internal__/input/input-presentation.component";
 
+function renderNumberInput(props, renderer = mount) {
```

Comment:
We don't need the second arg here as mount is the only renderer used

Reply:
right, removing it now